Sofa/Component/Mass/src/sofa/component/mass/MeshMatrixMass.inl

```diff
@@ -2190,9 +2191,9 @@ void MeshMatrixMass<DataTypes, GeometricalTypes>::addMToMatrix(const core::Mecha
 
     if(isLumped())
     {
-        for (size_t i=0; i<vertexMass.size(); i++)
+        for (unsigned int i = 0; i < sofa::helper::narrow_cast<unsigned int>(vertexMass.size()); ++i)
```

Comment:
is this narrow_cast really needed?
maybe use sofa::Size i    or sofa::Index  ?

Reply:
you're right. I think we can do better